dashboard/config/scripts/levels/CSD U3 AnimationsMulti defender enemies move.level

```diff
@@ -32,7 +32,6 @@
     "hide_view_data_button": "false",
     "debugger_disabled": "true",
     "hide_animation_mode": "false",
-    "show_d_pad": "false",
```

Comment:
A number of these levels have `show_d_pad` set to false, but you're removing that parameter and making the default true.  Do we need to also update these levels to use the new block?

Reply:
this param hasn't really been used. when it was, it was primarily hiding it on the desktop. For about a year, it did nothing at all.

The new default is "not visible on desktop, visible on the mobile share page"

I don't believe that the level builders ever set this flag considering the mobile share experience.